
All the 15 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
15 fits failed with the following error:
Traceback (most recent call last):
  File "F:\Anaconda\envs\conda_env\lib\site-packages\sklearn\model_selection\_validation.py", line 866, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "F:\Anaconda\envs\conda_env\lib\site-packages\sklearn\base.py", line 1389, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "F:\Anaconda\envs\conda_env\lib\site-packages\sklearn\pipeline.py", line 652, in fit
    Xt = self._fit(X, y, routed_params, raw_params=params)
  File "F:\Anaconda\envs\conda_env\lib\site-packages\sklearn\pipeline.py", line 560, in _fit
    self._validate_steps()
  File "F:\Anaconda\envs\conda_env\lib\site-packages\sklearn\pipeline.py", line 337, in _validate_steps
    raise TypeError(
TypeError: All intermediate steps should be transformers and implement fit and transform or be the string 'passthrough' 'SMOTENC(categorical_features=[2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 16,
                              17, 19, 20, 21, 22, 23, 24, 25, 26, 31],
        random_state=42, sampling_strategy='minority')' (type <class 'imblearn.over_sampling._smote.base.SMOTENC'>) doesn't
